src/openzeppelin/tests/access/test_accesscontrol.cairo

```diff
@@ -234,7 +251,15 @@ fn test_revoke_role_for_granted_role() {
     testing::set_caller_address(ADMIN());
 
     AccessControlImpl::grant_role(ref state, ROLE, AUTHORIZED());
+    testing::pop_log_raw(ZERO());
```

Comment:
this is to drop the granted event since we don't want to test it, right?

Reply:
Yes.